src/Flex.php

```diff
@@ -594,6 +594,18 @@ private function fetchRecipes(): array
             }
 
             if (isset($manifests[$name])) {
+                if (!empty($manifests[$name]['manifest']['conflict'])) {
+                    $lockedRepository = $this->composer->getLocker()->getLockedRepository();
+
+                    foreach ($manifests[$name]['manifest']['conflict'] as $conflictingPackageName => $constraint) {
+                        if ($conflictingPackage = $lockedRepository->findPackage($conflictingPackageName, $constraint)) {
+                            $this->io->writeError(sprintf('    <warning>Recipe for %s conflicts with %s:%s which should not be %s</>', $name, $conflictingPackageName, $conflictingPackage->getPrettyVersion(), $constraint));
```

Comment:
That's now a warning - all is fine, keep cool :)
I'd just remove the call personally. Nobody expects a recipe to come. Things work reverse: you install and you see what *might* come, without knowing beforehand.

Reply:
If verbose, without warning, ok for you?